web/sites/default/config/core.entity_form_display.node.information.default.yml

```diff
@@ -4,27 +4,29 @@ status: true
 dependencies:
   config:
     - field.field.node.information.field_body
+    - field.field.node.information.field_body_content
```

Comment:
Since we are moving this content type into information. We don't need to add the field to the information content type. (In fact, it might be confusing for migrators.)